package.json

```diff
@@ -48,6 +48,6 @@
     }
   },
   "lint-staged": {
-    "*.{js,jsx,ts,tsx}": "biome format --write"
+    "*.{js,jsx,ts,tsx}": "biome check --apply"
```

Comment:
this command runs formatter and linter 

https://biomejs.dev/reference/cli/#biome-check